etc/generate-envoy-config/pachyderm-services.libsonnet

```diff
@@ -123,6 +123,34 @@
       no_traffic_healthy_interval: '10s',
     },
   },
+  'pachd-download': {
+    internal_port: 1659,
+    service: 'pachd-proxy-backend',
+    routes: [
+      {
+        match: {
+          prefix: '/download',
+        },
+        route: {
+          cluster: 'pachd-download',
+          idle_timeout: '600s',
```

Comment:
why have such a long idle timeout here?

Reply:
Just paranoia about reading taking a long time.  I applied the same settings as with pachd-grpc.